plugins/woocommerce-admin/client/task-lists/fills/steps/location.tsx

```diff
@@ -91,6 +93,7 @@ const StoreLocation = ( {
 			},
 		} );
 
+		setSubmitting( false );
 		if ( ! isSettingsError ) {
```

Comment:
Not your work, but this code boggles my mind. As I understand it, the closure will capture the reference of isSettingsError before the onSubmit runs, and thus it will probably not work as intended (?)

Reply:
Good catch, I've removed usage of `isSettingsError` altogether. I think it was pretty useless 😆 https://github.com/woocommerce/woocommerce/pull/39613/commits/a585555d924f38dedf4e1c3f90e8716c61951fe3#diff-b57e737b7d66c46d56fd34bb695f74436b4a555d3cc4ab21d0e3675178855a1aR82-R83